tests/test_secretsmanager/test_secretsmanager.py

```diff
@@ -1950,3 +1951,23 @@ def test_update_secret_version_stage_dont_specify_current_stage(secret_arn=None)
         err["Message"]
         == f"The parameter RemoveFromVersionId can't be empty. Staging label AWSCURRENT is currently attached to version {current_version}, so you must explicitly reference that version in RemoveFromVersionId."
     )
+
+
+@mock_aws
+@pytest.mark.skipif(
+    not settings.TEST_DECORATOR_MODE, reason="Can't access the id manager in proxy mode"
+)
+def test_create_secret_custom_id(account_id, set_custom_id):
```

Comment:
Another non-blocking nitpick - the `account_id` fixture is only useful for tests that will also be run against AWS.

It's faster (and better readable IMO) to use the static `ACCOUNT_ID` directly.